app/util/codec_server.go

```diff
@@ -1,4 +1,4 @@
-package temporalutil
+package util
```

Comment:
Was hoping to avoid general util (https://dave.cheney.net/practical-go/presentations/qcon-china.html#_avoid_package_names_like_base_common_or_util) but not that big of a deal I suppose